src/App.tsx

```diff
@@ -7,38 +7,37 @@ import classNames from 'classnames';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
-import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [selectedUser, setSelectedUser] = useState<number | null>(null);
+  const [selectedPost, setSelectedPost] = useState<number | null>(null);
```

Comment:
`number and null` are strange types combinations, maybe selectedUser and selectedPost should be object instead of `number` then we can use `Post | null`
if selectedUser and selectedPost are ids then `null` shouldn't be here